### PR TITLE
added character limits

### PR DESCRIPTION
**Pull request by zach1502**
### Type

- [ ] Bugfix
- [ ] Enhancement
- [ ] New feature

### Description of the changes
title.
off to breaking things  :)

            _
               /`_>
              / /
              |/
          ____|    __
         |    \.-``  )    the hat doesnt look good...
         |---``\  _.'
      .-`'---``_.'
     (__...--``

### CODE REVIEW

File: cogs/general.py [hunk 1]
```diff
@@ -68,7 +68,13 @@ def __init__(self, bot):
...
         """
         author = ctx.message.author
-        if number > 1:
+        if number > 100000000000000000000000000000000000000000000:
+            await self.bot.say("...no...just why..? Keep the number below the total " +
+                               "(weight in grams) "+
+                               "of matter contained in the grand total of all " +
+                               "the stars in the Milky Way Galaxy. " +
+                               "(also called a Tredecillion or a septilliad")
+        elif number > 1 and number < 100000000000000000000000000000000000000000000:
```
Comment (KamiCreed) [inline cogs/general.py]: At least extract out the the magic number and put it as a constant above.

File: cogs/general.py [hunk 2]
```diff
@@ -154,8 +160,11 @@ def __init__(self, bot):
     @commands.command()
     async def lmgtfy(self, *, search_terms : str):
         """Creates a lmgtfy link"""
-        search_terms = escape_mass_mentions(search_terms.replace("+","%2B").replace(" ", "+"))
-        await self.bot.say("https://lmgtfy.com/?q={}".format(search_terms))
+        if len(search_terms) > 1000:
+            await self.bot.say("I think you know why I won't allow this...")
```
Comment (KamiCreed) [inline cogs/general.py]: At least put in brackets at the end `(Too long)` or something.

**Review by KamiCreed — CHANGES_REQUESTED** [review]

File: cogs/general.py (hunk 2 above)
Comment (KamiCreed) [reply cogs/general.py]: And also, does google have a limit of 1000 characters for searches?

Comment (Injabie3) [reply cogs/general.py]: I think similar to other PRs, this is to address Discord message content limits.

**Comment by KamiCreed** [comment]
I know I gave you suggestions to improve this, but ultimately, I don't think there's any point in putting limits here, since there's no optimization problems for the random number thing and if Imgur doesn't allow huge search terms, it'll probably error out. Basically what I'm saying is that we shouldn't place arbitrary limits if we can help it.

**Pull request closed without merging**